Source/m_menu.c

```diff
@@ -5050,6 +5114,7 @@ boolean M_Responder (event_t* ev)
 	    usegamma == 2 ? s_GAMMALVL2 :
 	    usegamma == 3 ? s_GAMMALVL3 :
 	    s_GAMMALVL4;
+	  gamma2 = 9; // 1.0f
```

Comment:
So, if we reset `gamma2` back to 9 when we hit F11, shouldn't we also reset `usegamma` back to 0 when using the fine-grained thermo in the menu?

Reply:
I'm thinking maybe we should remove the original `gammatable` and rename `gamma2` to `usegamma`?

Reply:
I think we still need the original gamma tables. Speedrunners sometimes position themselves by the change of pixel colors in specific wall textures, and we would take this opportunity from them for non-1 gamme values.